actors/common/src/main/java/org/sunbird/learner/actors/coursebatch/service/UserCoursesService.java

```diff
@@ -108,6 +109,18 @@ public void unenroll(String userId, String courseId, String batchId) {
     sync(updateAttributes, userCourses.getId());
   }
 
+  public Map<String, Object> getActiveUserCourses(String userId) {
+    Map<String, Object> filter = new HashMap<>();
+    filter.put(JsonKey.USER_ID, userId);
+    filter.put(JsonKey.ACTIVE, ProjectUtil.ActiveStatus.ACTIVE.getValue());
```

Comment:
Are you sure - it shud be JsonKey.STATUS - i believe

Reply:
if its confirmed fine.

Reply:
It's confirmed, the key is JsonKey.ACTIVE .